cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -79,18 +79,23 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 	if err != nil {
 		return fmt.Errorf("could not initialize a tracer: %w", err)
 	}
-
+	statusReporter := func(ev *component.StatusEvent) {
```

Comment:
you don't need this., nor the use of jtracer. The function `newServer` receives `component.TelemetrySettings` initialized by OTEL Collector. You can simply transform that into our internal `telemetry.Settings`, as it will contain all the required dependencies.

Reply:
Noted.

Reply:
Made the specified changes.